tooling/build-types-and-manifests.ts

```diff
@@ -44,8 +44,9 @@ addManifest('firefox-consumer', manifest => {
       strict_min_version: '112.0', // eslint-disable-line @typescript-eslint/naming-convention
     },
   };
+  delete manifest.background;
   manifest.background = {
-    ...manifest.background,
+    type: 'module',
     scripts: ['/js/service_worker/background.js'],
   };
```

Comment:
no need to delete `manifest.background`, as it's reassigned on the next line, so `delete manifest.background;` can be removed

Reply:
Oh - okay, I'll remove it now. thanks!